master/reference/calicoctl/resources/policy.md

```diff
@@ -65,6 +65,52 @@ the policy is enforced after connection tracking and any DNAT.
 See [Using Calico to Secure Host Interfaces]({{site.baseurl}}/{{page.version}}/getting-started/bare-metal/bare-metal)
 for how `doNotTrack` and `preDNAT` can be useful for host endpoints.
 
+The `types` field is useful when a policy specifies no ingress rules and you
+want that to mean that no ingress should be allowed, to each endpoint to which
+that policy applies, except as allowed by any other policies that apply to that
+endpoint.  You can express that by including `ingress` as one of the values of
+the `types` field - effectively saying "please apply this policy to ingress
+traffic, even though it looks like it should not."
+
+Similarly, when a policy specifies no egress rules and you want that to mean
+that no egress should be allowed ..., you can express that by including
+`egress` as one of the values of the `types` field.
+
+When `types` is not specified (or if it is an empty list), Calico treats the
+policy as applicable to ingress traffic (to the selected endpoints) if it has
+any ingress rules, or if it has neither ingress nor egress rules - with the
+latter case meaning to deny ingress by default - and to egress traffic (from
+the selected endpoints) only if it has any egress rules.
+
```

Comment:
The table that follows is not rendering properly in GitHub....I think something may be off here with your syntax. Take a look here at the syntax https://help.github.com/articles/organizing-information-with-tables/